db/schema.rb

```diff
@@ -71,7 +71,7 @@
     t.string   "overview_content_type"
     t.integer  "overview_file_size"
     t.datetime "overview_updated_at"
-    t.integer  "order"
+    t.integer  "order",                          default: 1
```

Comment:
@jasonoverby One trick I use for making commits that are granular enough that I can go back and pick stuff out is: `git add -p` instead of `git add -A` or whatever you might use to add everything in one chunk. You can get details for how to use it with `git add --help`.